test/src/test/java/org/corfudb/integration/CorfuReplicationReconfigurationIT.java

```diff
@@ -215,6 +224,108 @@ public void testActiveClusterReset() throws Exception {
         assertThat(remainingEntriesToSend).isEqualTo(0L);
     }
 
+    /**
+     * Event table can be cleared after processing the latest force sync event to avoid keeping older events.
+     *
+     */
+    @Test
+    public void testEventTableCleared() throws Exception {
+        String testStreamName = "Table001";
+        int batchSize = 5;
+
+        setupActiveAndStandbyCorfu();
+
+        Table<StringKey, IntValue, Metadata>  mapActive = corfuStoreActive.openTable(
+                NAMESPACE,
+                testStreamName,
+                StringKey.class,
+                IntValue.class,
+                Metadata.class,
+                TableOptions.builder().schemaOptions(
+                                CorfuOptions.SchemaOptions.newBuilder()
+                                        .setIsFederated(true)
+                                        .addStreamTag(ObjectsView.LOG_REPLICATOR_STREAM_INFO.getTagName())
+                                        .build())
+                        .build()
+        );
+
+        Table<StringKey, IntValue, Metadata>  mapStandby = corfuStoreStandby.openTable(
+                NAMESPACE,
+                testStreamName,
+                StringKey.class,
+                IntValue.class,
+                Metadata.class,
+                TableOptions.builder().schemaOptions(
+                                CorfuOptions.SchemaOptions.newBuilder()
+                                        .setIsFederated(true)
+                                        .addStreamTag(ObjectsView.LOG_REPLICATOR_STREAM_INFO.getTagName())
+                                        .build())
+                        .build()
+        );
+
+        // Write batchSize num of entries to active map
+        for (int i = 0; i < batchSize; i++) {
+            try (TxnContext txn = corfuStoreActive.txn(NAMESPACE)) {
+                txn.putRecord(mapActive, StringKey.newBuilder().setKey(String.valueOf(i)).build(),
+                        IntValue.newBuilder().setValue(i).build(), null);
+                txn.commit();
+            }
+        }
+        assertThat(mapActive.count()).isEqualTo(batchSize);
+        assertThat(mapStandby.count()).isZero();
+
+        startActiveLogReplicator();
+        startStandbyLogReplicator();
+        log.info("Replication servers started, and replication is in progress...");
+
+        while (mapStandby.count() != mapActive.count()) {
+            // Wait until data is fully replicated
```

Comment:
Do we need a timeout if the test keep on running in while loop?

Reply:
Yes the global limit is enforced but probably best to avoid the empty whiles so I've gone ahead and refactored